gene_splicer/utils.py

```diff
@@ -497,12 +563,14 @@ def generate_table_precursor_2(hivseqinr_resultsfile, filtered_file,
     return table_precursorfile
 
 
-def generate_proviral_landscape_csv(outpath):
+def generate_proviral_landscape_csv(outpath, is_hivintact):
     proviral_landscape_csv = os.path.join(outpath, 'proviral_landscape.csv')
     landscape_rows = []
 
     table_precursor_csv = os.path.join(outpath, 'table_precursor.csv')
     blastn_csv = glob.glob(
```

Comment:
This expression is very complicated. I generally try to avoid conditional expressions if they don't fit on one line. Adding a subscript at the end of a complicated expression can be difficult to trace back when reading.
Consider breaking this into three parts: choosing a glob target, doing the glob, taking the first match.

Reply:
Absolutely :+1: